decidim-core/spec/factories.rb

```diff
@@ -7,3 +7,4 @@
 require "decidim/meetings/test/factories"
 require "decidim/accountability/test/factories"
 require "decidim/system/test/factories"
+require "decidim/initiatives/test/factories"
```

Comment:
Do we really need to add this dependecy to decidim core?
Our objective is to get rid of `accountability` and `meetings` not to add more dependencies :smile: